inc/namespace.php

```diff
@@ -196,3 +198,77 @@ function enqueue_scripts() {
 	// Print queued scripts.
 	print_head_scripts();
 }
+
+/**
+ * Schedule index maintenance daily.
+ */
+if( ! wp_next_scheduled( __NAMESPACE__ . '\\index_maintenance' ) ) {
+	wp_schedule_event( time(), 'daily',  __NAMESPACE__ . '\\index_maintenance' );
+}
+
+
+/**
+ * Delete old Analytics indexes in Elasticsearch.
+ */
+function delete_old_indexes() : ?array {
+	// Index age in days.
+	$duration = apply_filters( 'altis.analytics.index_age', 7 );
```

Comment:
Lets cast this to an integer, and we can make the filter name a bit more descriptive too:
```suggestion
	$duration = (int) apply_filters( 'altis.analytics.max_index_age', 7 );
```